common/lib/capa/capa/responsetypes.py

```diff
@@ -709,6 +709,12 @@ def setup_response(self):
         self.correct_choices = set([choice.get(
             'name') for choice in correct_xml])
 
+        incorrect_xml = self.xml.xpath('//*[@id=$id]//choice[@correct="false"]',
+                                     id=self.xml.get('id'))
+
+        self.incorrect_choices = set([choice.get(
+            'name') for choice in incorrect_xml])
```

Comment:
Just fix your added stuff to conform to good style. Capa is one of the
oldest parts of our codebase and has some of the absolute worst code
quality.

On Tue, Jun 23, 2015 at 2:13 PM, Colin-Fredericks <notifications@github.com>
wrote:

> In common/lib/capa/capa/responsetypes.py
> <https://github.com/edx/edx-platform/pull/8574#discussion_r33072826>:
>
> > @@ -709,6 +709,12 @@ def setup_response(self):
> >          self.correct_choices = set([choice.get(
> >              'name') for choice in correct_xml])
> >
> > +        incorrect_xml = self.xml.xpath('//*[@id=$id]//choice[@correct="false"]',
> > +                                     id=self.xml.get('id'))
> > +
> > +        self.incorrect_choices = set([choice.get(
> > +            'name') for choice in incorrect_xml])
>
> It's broken in a way identical to line 706 (correct_xml = ...), since I
> copied it from there. I can fix that one too if you like.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/edx/edx-platform/pull/8574/files#r33072826>.
>